config/ralscfg.go

```diff
@@ -33,6 +33,7 @@ type RalsCfg struct {
 	RemoveExpired            bool
 	RALsMaxComputedUsage     map[string]time.Duration
 	RALsBalanceRatingSubject map[string]string
+	RALsMaxIncrements        int
```

Comment:
Since we have now rals in separate config, please rename all it's config variables without the RALs prefix